src/ansys/dpf/core/outputs.py

```diff
@@ -56,7 +57,17 @@ def get_data(self):
         elif type_output == "vector<int32>":
             type_output = types.vec_int
 
-        return self._operator.get_output(self._pin, type_output)
+        type_output_derive_class = self._spec.name_derived_class
+
+        if type_output_derive_class != "":
+            out_type = [
+                type_tuple
+                for type_tuple in self._operator._type_to_output_method
+                if type_output_derive_class in type_tuple
+            ]
+            return out_type[0][0](self._operator.get_output(self._pin, type_output))
```

Comment:
This should be evaluated in:

MeshInfo(self._operator.get_output(self._pin, dpf.types.generic_data_container))